app/src/main/kotlin/com/ayatk/biblio/ui/util/helper/DataBindingHelper.kt

```diff
@@ -45,4 +50,47 @@ object DataBindingHelper {
       this.addView(frameLayout)
     }
   }
+
+  @JvmStatic
+  @BindingAdapter("addRankingItems")
+  fun LinearLayout.addRankingItems(rankings: List<Ranking>) {
+    this.removeAllViews()
+    val inflater = this.context.getSystemService(Context.LAYOUT_INFLATER_SERVICE) as LayoutInflater
+    rankings.map {
+      val rankingItem = inflater.inflate(R.layout.view_ranking_top_item, null)
+      val rank = rankingItem.findViewById<ImageView>(R.id.rank)
+      val rankText = rankingItem.findViewById<TextView>(R.id.rank_text)
+      // ランキングのイメージ
+      when (it.rank) {
+        1 -> {
+          rankText.visibility = View.GONE
+          rank.setColorFilter(ContextCompat.getColor(context, R.color.gold))
+        }
+        2 -> {
+          rankText.visibility = View.GONE
+          rank.setColorFilter(ContextCompat.getColor(context, R.color.silver))
+        }
+        3 -> {
```

Comment:
**[detekt]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Report magic numbers. Magic number is a numeric literal that is not defined as a constant and hence it's unclear what the purpose of this number is. It's better to declare such numbers as constants and give them a proper name. By default, -1, 0, 1, and 2 are not considered to be magic numbers.